game/game/gamescript.cpp

```diff
@@ -1277,6 +1277,14 @@ uint32_t GameScript::rand(uint32_t max) {
   return uint32_t(randGen())%max;
   }
 
+Npc* GameScript::globalOther() {
```

Comment:
Having `globalOther` and have no `globalSelf`, `globalVictum`, etc doesn't fill quite right.

Maybe you can create `Npc* GameScript::findNpc(const symbol*)` instead?